modules/dkimVerifier.jsm

```diff
@@ -1209,7 +1233,23 @@ var Verifier = (function() {
 
```

Comment:
Don't know if you let it out intentionally, but I also added a check to ensure the signature algo matches the key type (the `DKIM_SIGERROR_KEY_MISMATCHED_K` error).
Not strictly needed as verification should fails then, just makes it more clear why it fails.

I would also be fine to not include the check here.

Reply:
Actually I missed that, I'll add the check, too

Reply:
I implemented the DKIM_SIGERROR_KEY_MISMATCHED_K error now :)